server/src/services/server.service.spec.ts

```diff
@@ -177,8 +178,16 @@ describe(ServerService.name, () => {
     });
   });
 
+  vi.mock('fs', async () => {
+    const actual = (await vi.importActual<typeof fs>('fs'))!;
+    return {
+      ...actual,
+      statSync: vi.fn().mockReturnValue({ size: 500 }),
+    };
+  });
```

Comment:
This is now how you should be doing that. You should be mocking repo methods if you have to